### ISSUE TITLE
Clone tool ignores source's opacity

### ISSUE THREAD

**Issue by rj-rl** [bgrabitmap/lazpaint#449]
**Severity**
High

**Defect descirption**
Clone tool does not respect the source's opacity

**Steps to reproduce**
- Pick any drawing tool such as a brush 
- Set opacity to anything below 255, the lower the value, the more pronounced the effect
- Draw any transparent shape
- Use clone tool to clone the shape elsewhere on the same layer

**Expected result**
The cloned shape is exactly the same as the source shape

**Actual result**
The cloned shape's color lacks some (or all) transparency

**Environment**
Windows 10, build 19042.1415, version 20H2
LazPaint 7.1.6 (64-bit)

https://user-images.githubusercontent.com/84194921/146567283-63cadfec-3b5e-4fbd-bf39-d30b6da8570f.mp4

**Comment by circular17** [comment]
Yes this is due to the clone brush drawing each time mouse moves. 

An option to clone only once each pixel could be added
